back/go.mod

```diff
@@ -7,22 +7,31 @@ require (
 	github.com/ahmetb/go-linq/v3 v3.1.0
 	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751
 	github.com/cpuguy83/go-md2man/v2 v2.0.0 // indirect
-	github.com/facebookincubator/ent v0.1.2
+	github.com/facebook/ent v0.4.3
```

Comment:
von 0.1.2 zu 0.4.3 :sweat_smile: da würde es sich bestimmt lohnen in die release notes zu schauen